public-safety-uss/swagger.yaml

```diff
@@ -38,7 +38,7 @@ info:
     UNILATERAL TERMINATION OF THIS AGREEMENT.
 
 host: uss.defined.url
-basePath: /pubsafe
```

Comment:
From my understanding the gh issue asks only to relax base path for USS API.    
I prefer that the pubsafe be defined in advance since it can never  be null becuase the default API is USS-API which can be null/empty